# slam_ekf_config.yaml
length: 0.5
width: 0.4
wheel_radius: 0.1
initial_state: [-2.15, -6.906, 0, 0, 0, 0]

#LANDMARK PARAMETERS
landmark_treshold: 0.5
landmark_accuracy: 0.2

#DB SCAN PARAMETERS
eps: 0.3
min_samples: 3

Q_states:
- [0.05, 0, 0, 0, 0, 0]
- [0, 0.05, 0, 0, 0, 0]
- [0, 0, 0.01, 0, 0, 0]
- [0, 0, 0, 0.001, 0, 0]
- [0, 0, 0, 0, 0.001, 0]
- [0, 0, 0, 0, 0, 0.0005]

P0_states:
  - [0.5, 0, 0, 0, 0, 0]
  - [0, 0.5, 0, 0, 0, 0]
  - [0, 0, 0.3, 0, 0, 0]
  - [0, 0, 0, 0.1, 0, 0]
  - [0, 0, 0, 0, 0.1, 0]
  - [0, 0, 0, 0, 0, 0.05]

R_imu:
  - [0.01, 0, 0]
  - [0, 0.01, 0]
  - [0, 0, 0.1]
R_encoder:
  - [0.005, 0, 0]
  - [0, 0.005, 0]
  - [0, 0, 0.0001]

R_lidar:
  - [0.1, 0]
  - [0, 0.1 ]

